README

```diff
@@ -93,7 +93,7 @@ First, build Pachi with DCNN support:
   dependencies.
 - Edit Makefile, set DCNN=1, point it to where caffe is installed and build.
 
-Install dcnn files in current directory where pachi will run.
+Install dcnn files in Pachi's data directory.
 Detlef Schmicker's 54% dcnn can be found at:  
   http://physik.de/CNNlast.tar.gz
 
```

Comment:
But will the reader have any idea what _is_ Pachi's data directory? We should either explain that (together with possibility of using the current directory) and use it consistently (also for pattern files etc.), explain that (and that it is a fallback to the current directory behavior) and leave the other uses as is, or just stay silent about it as an undocumented packging feature (least preferred).